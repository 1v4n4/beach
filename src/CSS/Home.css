.down {
  position: fixed;
  bottom: 0;
  text-align: center;
  color: rgb(124, 121, 121);
}

.down-link {
  color: rgb(124, 121, 121);
}

down-link:hover {
  color: #333;
}