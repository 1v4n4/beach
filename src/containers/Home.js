import React from 'react';
import BeachList from '../components/BeachList';

const Home = () => (
  <>
    <h1>Home</h1>
    <p>Lorem ipsummmmmm</p>

    <BeachList />

  </>
);

export default Home;
