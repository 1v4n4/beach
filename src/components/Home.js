import React from 'react';
import { FaUmbrellaBeach } from 'react-icons/fa';
import BeachList from './BeachList';
import '../CSS/Home.css';

const Home = () => (
  <>
    <h1 className="text-center my-3">
      <FaUmbrellaBeach className="pb-1 pe-1" />
      Cal Beach
    </h1>
    <p className="text-center fs-4 font-italic mx-5">Explore the coast of California and find your perfect place</p>

    <BeachList />

  </>
);

export default Home;
