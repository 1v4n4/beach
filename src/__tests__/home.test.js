import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from '../components/Home';
import createTestStore from './store';

describe('Homepage', () => {
  let store;
  beforeEach(() => {
    store = createTestStore();
  });

  test('should match with snapshot', () => {
    const jsx = (
      <Provider store={store}>
        <BrowserRouter>
          <Home />
        </BrowserRouter>
      </Provider>
    );
    const tree = (() => renderer.create(jsx).toJSON());
    expect(tree).toMatchSnapshot();
  });
});
