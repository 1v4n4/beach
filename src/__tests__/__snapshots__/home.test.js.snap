// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage should match with snapshot 1`] = `[Function]`;
